src/content/reference/app-defaulting-validation/index.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Defaulting and Validation of App CRs
+description: How defaulting and validation of app CRs is implemented by app-admission-controller
+weight: 110
+type: page
+last_review_date: 2020-12-16
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-batman
+---
+
+## Overview
+
+For Giant Swarm releases using app-operator version 3.0.0 and upwards the
+defaulting and validation logic of [App CRs](/reference/cp-k8s-api/apps.application.giantswarm.io/) is enabled. This logic is provided by
```

Comment:
IMO using `relref` to spot dead links before deploying the site is still a good idea. I tested this locally, and this time the syntax and reference is correct ;)

```suggestion
defaulting and validation logic of [App CRs]({{< relref "/content/reference/cp-k8s-api/apps.application.giantswarm.io.md" >}}) is enabled. This logic is provided by
```

Reply:
Fine with me :)

Reply:
```
ERROR 2021/01/18 13:15:29 [en] REF_NOT_FOUND: Ref "/content/reference/cp-k8s-api/apps.application.giantswarm.io.md": "/docs/src/content/reference/app-defaulting-validation/index.md:14:46": page not found
Error: Error building site: logged 1 error(s)
```

https://app.circleci.com/pipelines/github/giantswarm/docs/1930/workflows/97999bd9-43ed-4ef7-b353-fd7b552f7d21/jobs/4640

@ubergesundheit The build failed when I updated the branch. So I've reverted back to the previous link. Do you know why it failed?

Reply:
Sorry for breaking your build (again) :(

Not your fault, because the master branch moved the document to `/content/reference/management-api/apps.application.giantswarm.io.md`.

So the build error was good after all to show that the old link pointing to `/reference/cp-k8s-api/apps.application.giantswarm.io/` was not correct any more.

So this suggestion should work (and break builds for people in the future who move the linked document)

```suggestion
defaulting and validation logic of [App CRs]({{< relref "/content/reference/management-api/apps.application.giantswarm.io.md" >}}) is enabled. This logic is provided by
```

Reply:
> Sorry for breaking your build (again) :(

Thanks @ubergesundheit I'd missed the renaming. Great that the build caught it!